src/clj/ataru/applications/application_store.clj

```diff
@@ -296,14 +296,28 @@
        first
        :virkailija_oid))
 
+(def chars-to-filter [\u0000])
+
+(defn filter-chars [coll chars]
+  (apply str (remove #((set chars) %) coll)))
+
+(defn filter-answer-values [value]
+  (if (string? value)
+    (clojure.string/replace value "\u0000" "")
+    (if (vector? value)
+      (into [] (map filter-answer-values value))
+      (into '() (map filter-answer-values value)))))
```

Comment:
Tää kääntää listan järjestyksen, joka ei oo hyvä. Tuo `(into [] ...)` pitäis olla turvallinen tässä kohtaa myös muillekkin ei-merkkijonoille kun vektoreille.